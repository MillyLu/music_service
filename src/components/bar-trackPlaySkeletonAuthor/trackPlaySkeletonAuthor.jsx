import * as Styled from './styles';

export function TrackPlaySkeletonAuthor() {
    return(
        <Styled.TrackPlayAuthor>
           <Styled.SkeletonTrackAlbum></Styled.SkeletonTrackAlbum>
        </Styled.TrackPlayAuthor>
    )
}