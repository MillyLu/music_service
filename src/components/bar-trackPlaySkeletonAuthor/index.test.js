/* eslint-disable no-undef */
import * as ShallowRenderer from 'react-shallow-renderer';
// import { ShallowRenderer, createRenderer } from 'react-shallow-renderer';
import { TrackPlaySkeletonAuthor } from './trackPlaySkeletonAuthor';



describe('<TrackPlaySkeletonAuthor/>', () => {
        it('should render successfully and match snapshot', () => {
            const shallowRenderer = ShallowRenderer.createRenderer();
            const renderer = shallowRenderer;
            renderer.render(<TrackPlaySkeletonAuthor />);
            const result = renderer.getRenderOutput();
            expect(result).toMatchSnapshot();
          })
      });