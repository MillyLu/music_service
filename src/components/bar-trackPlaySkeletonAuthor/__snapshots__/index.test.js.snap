// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrackPlaySkeletonAuthor/> should render successfully and match snapshot 1`] = `
<styled.div>
  <styled.div />
</styled.div>
`;
